test/yargs.js

```diff
@@ -311,6 +311,30 @@ describe('yargs dsl tests', function () {
   })
 
   describe('locale', function () {
+    it('returns locale if locale is called with no arguments', function () {
+      yargs.locale().should.equal('en')
```

Comment:
This assertion changes with `y18n@3.1.0`:

```js
yargs.locale().should.equal('en_US')
```

Should probably `loadLocale('en_US.UTF-8')` at beginning of this test.